test/format/format.h

```diff
@@ -90,6 +90,9 @@
 #define STR(s) #s
 #define XSTR(s) STR(s)
 
+/* Session configuration to enable prefetch. */
+#define PREFETCH_SESSION_CFG_ON "prefetch=(enabled=true)"
```

Comment:
Minor suggestion: Since this is a session configuration, I'd prefer `SESSION_PREFETCH_CFG_ON` just to put the session as the highest level thing.

Reply:
Renamed in 7b0a87991222fc6f6f2caa9142e18d281bb6af0f